src/components/layouts/NavigationBar/index.tsx

```diff
@@ -0,0 +1,3 @@
+import NavigationBar from './NavigationBar';
+
+export default NavigationBar;
```

Comment:
I am not sure about this.. The barrel file is to export everything in the NavigationBar folder

```
export NavigationBar from './NavigationBar`
export NavigationButton from './NavigationButton`
```

For example if we were to have a navigationButton in here

Reply:
We need to make another folder for it. In the folder, we can set up styles, test, and inner components used.
Reason why I added copy-paste index.tsx is to prevent ugly import in other files.

eg:
`import NavigationBar from '@components/layout/NavigationBar/NavigationBar';`

Reply:
yup disregard my comment here actually you are right
Also should we have the barrel as index.ts or `index.tsx` not that it really matters here